README.md

```diff
@@ -222,6 +223,9 @@ Modules or frameworks for virtual machines that run in the Cosmos SDK
 * [Cosmos Discord Faucet](https://github.com/0x4139/cosmos-discord-faucet) - A configurable discord faucet for Cosmos SDK blockchains.
 * [Cosmos Discord Bot](https://github.com/okp4/discord-bot) - A general-purpose Discord bot for Cosmos SDK blockchains.
 
+### Indexers
```

Comment:
```suggestion
### Indexers

```

Reply:
@julienrbrt 

Happy new year 😄 
Thanks for your comment and I applied this.